### DISCUSSION TITLE
How to get IP address of clients without enabling DHCP

### DISCUSSION THREAD

**Discussion by laurentftech** [AdguardTeam/AdGuardHome#4062] [Q&A]
Hi. 
I do not succeed to get the IP adresses of all network clients. My AGH is on a NAS behind my Netgear router which is serving as DHCP server (address 10.0.0.1).
I tried to activate private rDNS resolver with the following addresses :

10.0.0.1
[/0.0.10.in-addr.arpa/]10.0.0.1
[/ip6.arpa/]10.0.0.1
[/local/]10.0.0.1

I have checked "use private reverse DNS resolvers" and "Enable reverse resolving of client's IP addresses".

All the logs coming from devices which have not DNS set to AGH IP are indicated as coming from the router (10.0.0.1). 

Could you please give me some guidance how to set it up correctly ? 
Thank you.

**Comment by gspannu** [comment]
Have you tried changing the name of the _local_domain_name_ parameter in the AdGuardHome.yaml file?

This should be the same as what you have defined in your NetGear router? 
The default value of this param is lan, but it should actually be _domain name specified in your NeGear router_.
local_domain_name: local <or something>
